shell/apps/common/config.js

```diff
@@ -1,6 +1,6 @@
 // establish Debug level before using logFactory
 import Xen from '../../components/xen/xen.js';
-Xen.Debug.level = ((new URL(document.location)).searchParams.has('log')) ? 2 : 0;
+Xen.Debug.level = ((new URL(document.location)).searchParams.has('log')) ? 3 : 0;
```

Comment:
Just making sure you want this in this PR?

Reply:
Nope, that was a mistake. Thanks!